distribution/asf-release-process-guide.md

```diff
@@ -248,10 +259,16 @@ must be tagged properly to make this script working. See the above [Release note
 Once the release branch is good for an RC, you can build a new tag with:
 
 ```bash
-$ mvn -DreleaseVersion=0.17.0 -DdevelopmentVersion=0.18.0-SNAPSHOT -Dtag=druid-0.17.0-rc3 -DpushChanges=false clean release:clean release:prepare
+$ mvn -Pwebsite-docs -DreleaseVersion=0.17.0 -DdevelopmentVersion=0.18.0-SNAPSHOT -Dtag=druid-0.17.0-rc3 -DpushChanges=false clean release:clean release:prepare
```

Comment:
does this do the right thing? The website-docs profile only has the website module, so will it only bump website version or is it additive?

Reply:
yeah. I verified it by running `mvn -Pwebsite-docs help:evaluate -Dexpression=project.modules`